mantle/platform/conf/conf.go

```diff
@@ -398,6 +398,49 @@ func MergeAllV34exp(confObjs []*Conf) (*UserData, error) {
 	return userData, nil
 }
 
+// Config is compressed and added to another via data url
+func ReplaceAndMaybeCompress(config string) (string, error) {
```

Comment:
The name is a bit misleading: "replace" is an implementation detail and we don't always do it.  Maybe `MaybeCompressConfig`?